client.go

```diff
@@ -443,6 +489,76 @@ func (client *Client) CfInfo(key string) (map[string]int64, error) {
 	return ParseInfoReply(redis.Values(conn.Do("CF.INFO", key)))
 }
 
+// TdCreate - Allocate the memory and initialize the t-digest
+func (client *Client) TdCreate(key string, compression int64) (string, error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	return redis.String(conn.Do("TDIGEST.CREATE", key, compression))
+}
+
+// TdReset - Reset the sketch to zero - empty out the sketch and re-initialize it
+func (client *Client) TdReset(key string) (string, error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	return redis.String(conn.Do("TDIGEST.RESET", key))
+}
+
+// TdAdd - Adds one or more samples to a sketch
```

Comment:
Suggest to change signature from `TdAdd(key string, samples map[string]float64) (string, error)` to `TdAdd(key string, samples map[float64]float64) (bool, error)`
Given that the commands replies with:

```
 OK on success, error otherwise 
 ```
https://oss.redislabs.com/redisbloom/master/TDigest_Commands/#tdigestadd